src/main/java/org/snomed/snowstorm/rest/ConceptController.java

```diff
@@ -273,6 +277,21 @@ public ConceptView findBrowserConcept(
 		return ControllerHelper.throwIfNotFound("Concept", concept);
 	}
 
+	@ApiOperation(value = "View the history of a Concept.")
+	@RequestMapping(value = "/browser/{branch}/concepts/{conceptId}/history", method = RequestMethod.GET, produces = {"application/json", "text/csv"})
```

Comment:
text/csv is not needed here and can be a testing headache, could you remove please?